### ISSUE TITLE
Tab separator

### ISSUE THREAD

**Issue by cbf305** [herbstluftwm/herbstluftwm#1421]
I really need to read the commit log more often, I just found out tabs were added and have been playing around with them.  It's a very nice feature, thanks for adding it!

Would it be possible to get a tab separator?  It would give the brain a break point to visually separate items.

I was thinking something like:
```
theme.tab_separator_color
theme.tab_separator_width
theme.tab_separator_height
```

Color and width are obvious, but I think it would be nice to have a height to this one.  Maybe it could default to title height minus 4 so it would look like it's floating between the top and bottom of the tab bar, sort of like the tab separator in the Chrome browser.  It would be just enough to give visual distinction to the tabs.  It would make sense to only show the separator with three or more tabs and only when the tab is not adjacent to the active tab as that is typically a different color.

Here is just a quick sample I threw together.  
![screenshot_2022-01-19_12-56-25](https://user-images.githubusercontent.com/2612500/150187662-f2d9a8ce-9c82-4df0-89db-615fe0023d82.png)

Thanks!

**Comment by t-wissmann** [comment]
Yes, I also thought about these separators, so they will come at some point :)

Regarding new features, you don't necessarily need to read the commit log. New features are also announced in the [NEWS file](https://herbstluftwm.org/news.html)

**Comment by cbf305** [comment]
This is good to hear.  I'm glad I wasn't the only one thinking of the separator :-)

Thanks for the reminder.  I always forget about the news page on the web site, however, I like to read the commit log because I find the commit messages and often the discussion about a feature or patch help me get the most out of hlwm.

Thanks!

**Comment by t-wissmann** [comment]
I forgot to answer here: the git version of hlwm currently have a css-style theming engine which makes it easy to define tab separators:
![rezlooks-green-tab-separator](https://github.com/herbstluftwm/herbstluftwm/assets/9048813/3dafd6d5-0508-4fbd-b523-6ecf530423b6)

The css file in use is here: https://github.com/t-wissmann/dotfiles/blob/master/config/herbstluftwm/themes/rezlooks-green.css
